test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1500,6 +1500,7 @@ def revert_mom(self, mom, force=False):
             mom.signal('-HUP')
             self.server.expect(NODE, a, id=mom.shortname + '[0]', interval=1)
         else:
+            self.server.manager(MGR_CMD_CREATE, NODE, None, mom.shortname)
```

Comment:
Remove the None and say id=mom.shortname

Reply:
Ok, I'll update.